common/persistence/tests/cassandra_test.go

```diff
@@ -135,3 +136,40 @@ func TestCassandraTaskQueueTaskSuite(t *testing.T) {
 	s := NewTaskQueueTaskSuite(t, taskQueueStore, testData.Logger)
 	suite.Run(t, s)
 }
+
+func TestCassandraVisibilityPersistence(t *testing.T) {
+	s := &VisibilityPersistenceSuite{
+		TestBase: persistencetests.NewTestBaseWithCassandra(&persistencetests.TestBaseOptions{}),
+	}
+	suite.Run(t, s)
+}
+
+// TODO: Merge persistence-tests into the tests directory.
```

Comment:
Do we need this anymore?

Reply:
Yes. I did not delete the tests under persistence-tests dir.

Reply:
May be it is time address this TODO, after you merge this PR.